### PR TITLE
Increase spacing between cards

### LINKED ISSUE #506: Add more space between card stacks
**Describe the bug**
When a card stack has three or more cards, it is hard to tell the difference between it and the adjoining stack.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by maimoona-bashir**
fixed bugs for cards overlapping

Fixes #506

#### Overview of changes

- Added margins around the stacks

Reviewer: @johnanvik

### CODE REVIEW

File: src/components/Modals/RulesModal.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
                     <p style="text-align: center; font-size: 24px;"><b>Playing Cards</b></p>
                     <li><h5>Instruction</h5>
-                      <div>
+                      <div >
```
Comment (johnanvik) [inline src/components/Modals/RulesModal.vue:31]: It looks like all of these changes are just white space. Rever the file back to the original.

File: src/components/Modals/RulesModal.vue [hunk 2]
```diff
@@ -93,8 +93,10 @@
                         here.</a>
                       </div>
                     </li>
+                    <!-- changes -->
```
Comment (johnanvik) [inline src/components/Modals/RulesModal.vue:96]: Unnecessary comment.

File: src/components/SharedComponents/Playfield.vue [hunk 3]
```diff
@@ -6,12 +6,13 @@
         <h3 style="text-align: left; margin-left: 40px" :style="playfieldTextColour()">playerStacks() {</h3>
       </div>
     </div>
-    <div class="row">
-        <div class="stacks col-lg-3 col-md-4 col-sm-6" v-for="stack in stacks">
+    <div class="row my-4">
```
Comment (johnanvik) [inline src/components/SharedComponents/Playfield.vue:9]: If you changed the class, shouldn't there also be a style file update in this commit?

**Comment by maimoona-bashir** [comment]
Hi,
I removed the void space between div, but i guess i have pulled the wrong
request. i don't know how to undo it.
Regards,
Maimoona Bashir

File: src/components/Modals/RulesModal.vue (hunk 1 above)
Comment (maimoona-bashir) [reply src/components/Modals/RulesModal.vue]: fixed

File: src/components/SharedComponents/Playfield.vue (hunk 3 above)
Comment (maimoona-bashir) [reply src/components/SharedComponents/Playfield.vue]: Resolved

File: src/components/Modals/RulesModal.vue (hunk 2 above)
Comment (maimoona-bashir) [reply src/components/Modals/RulesModal.vue]: Fixed

**Pull request merged**